src/main/java/com/harrison/controllers/SkillController.java

```diff
@@ -27,6 +35,38 @@
 
     @RequestMapping(value = "/{name}", method = RequestMethod.GET)
     public ResponseEntity<Skill> getSkill(@PathVariable("name") String name) {
+        TransactionStatus trans = transactionManager.getTransaction(new TransactionDefinition() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "trans". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "trans" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AS1481)